config/webpacker.yml

```diff
@@ -4,7 +4,7 @@ default: &default
   source_path: app/javascript
   source_entry_path: packs
   public_root_path: public
-  public_output_path: packs
```

Comment:
Without this it's difficult (if not impossible) for the reverse proxy to determine the correct app to serve in development.